_posts/2022-11-30-trino-summit-2022-comcast-recap.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Elevating Data Fabric to Data Mesh: solving data needs in hybrid data lakes"
+author: "Sajuman Joseph, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/comcast.jpg
+---
+
+Tune in, for the next [post in the Trino Summit 2022 recap series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). In this post, we're joining "Saj" from
```

Comment:
If you're gonna write "Saj" in quotes, it should be as Sajuman "Saj" Joseph. Otherwise, just Saj without quotes is fine. I'd probably lean towards doing the latter.